### DISCUSSION TITLE
Lost access to owner permission wallet of weight2 and a threshold of 2

### DISCUSSION THREAD

**Discussion by Matarwy** [tronprotocol/tips#675] [General]
in short, on my main wallet there is a threshold of 2, and any changes and operations require confirmation from YLRZ, where I lost access, and my main CPBA wallet has a weight of 1

I have a wallet TDGZ4GfvCRe1d8oksj8fBD77ZHw4bkCPBA (there is a private key), on which the last funds on it are worth weight 1, then there is a connected wallet TEUNJ2Ugba3ZQWgNxiTS7f3EfMBTTuYLRZ with weight 2 (there is no private key), that is, to carry out any transfer, a signature-confirmation with this wallet is required. I also have 3 wallets TXMg6GLMv5c6MX88yRs19Lq1p2vmRaiDY3 (there is a private key), their main wallet is TDGZ4GfvCRe1d8oksj8fBD77ZHw4bkCPBA, our task is to transfer money from my wallet TDGZ4GfvCRe1d8oksj8fBD77ZHw4bkCPBA to TXMg6G LMv5c6MX88yRs19Lq1p2vmRaiDY3 this task will be a constant, because there will be a variable signature with TEUNJ2Ugba3ZQWgNxiTS7f3EfMBTTuYLRZ

of course the smart contract itself is open TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t

and here are my permissions:

и вот мои разрешения MacBook-Pro-Macbook:tron-permission-update macbook$ node checkPermissions.cjs

Текущие разрешения: {

"address": "41243030a9b2ff560506d07b55d9daf70f42fd2262",

"balance": 259173491,

"create_time": 1685463957000,

"latest_opration_time": 1721067069000,

"allowance": 546670,

"latest_withdraw_time": 1696948017000,

"latest_consume_time": 1695679533000,

"latest_consume_free_time": 1721061885000,

"net_window_size": 28800000,

"net_window_optimized": true,

"account_resource": {

"latest_consume_time_for_energy": 1721064039000,

"energy_window_size": 28800000,

"energy_window_optimized": true

},

"owner_permission": {

"permission_name": "owner",

"threshold": 2,

"keys": [

{

"address": "41243030a9b2ff560506d07b55d9daf70f42fd2262",

"weight": 1

},

{

"address": "41316452b34adfe59cea9542b10a1cdcf2103d76ff",

"weight": 2

}

]

},

"active_permission": [

{

"type": "Active",

"id": 2,

"permission_name": "active",

"threshold": 2,

"operations": "0004000000000000000000000000000000000000000000000000000000000000",

"keys": [

{

"address": "41243030a9b2ff560506d07b55d9daf70f42fd2262",

"weight": 1

},

{

"address": "41316452b34adfe59cea9542b10a1cdcf2103d76ff",

"weight": 2

}

]

}

],

"frozenV2": [

{},

{

"type": "ENERGY"

},

{

"type": "TRON_POWER"

}

],

"unfrozenV2": [

{

"type": "ENERGY",

"unfreeze_amount": 50000000,

"unfreeze_expire_time": 1707253125000

}

],

"assetV2": [

{

"key": "1004977",

"value": 8888880000

},

{

"key": "1004978",

"value": 180

},

{

"key": "1004920",

"value": 8888888

},

{

"key": "1004975",

"value": 20000

},

{

"key": "1004952",

"value": 157400

},

{

"key": "1004950",

"value": 29980

},

{

"key": "1005026",

"value": 8888880000

},

{

"key": "1004992",

"value": 10000

},

{

"key": "1004990",

"value": 26640

}

],

"free_asset_net_usageV2": [

{

"key": "1004977",

"value": 0

},

{

"key": "1004978",

"value": 0

},

{

"key": "1004920",

"value": 0

},

{

"key": "1004975",

"value": 0

},

{

"key": "1004952",

"value": 0

},

{

"key": "1004950",

"value": 0

},

{

"key": "1005026",

"value": 0

},

{

"key": "1004992",

"value": 0

},

{

"key": "1004990",

"value": 0

}

],

"asset_optimized": true

}

**Comment by onm798** [comment]
Sorry for your loss, but there is nothing can help but to find the private-key of YLRZ.
